sentences/fr/vacuum_HassVacuumStart.yaml

```diff
@@ -0,0 +1,8 @@
+language: fr
+intents:
+  HassVacuumStart:
+    data:
+      - sentences:
+          - "(démarre|lance) [<le>]{name}"
```

Comment:
Same comment as for HassSetPosition : Verbs should be expansion rules and support infinitive forms